src/hooks/chain/useCreditManagerConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate'
+import useSWR from 'swr'
+import { getUrl } from 'utils/url'
+import useStore from 'store'
+import { getCurrentChainId } from 'utils/getCurrentChainId'
+import chains from 'chains'
+
+export default function useCreditManagerConfig() {
+  const chainId = getCurrentChainId()
+  const baseConfig = chains[chainId]
+
+  return useSWR(
```

Comment:
You can use an immutable call; these settings will not change during a user's session, so please make sure you only fetch it once via the immutable call.
```suggestion
  return useSWRImmutable(
```